verification/curator-service/api/openapi/openapi.yaml

```diff
@@ -0,0 +1,262 @@
+          in: query
+          description: The pages of sources to skip before starting to collect the result set
+          required: false
+          schema:
+            type: integer
+            format: int32
+            minimum: 1
+            default: 1
+        - name: limit
+          in: query
+          description: The number of sources to return
+          required: false
+          schema:
+            type: integer
+            format: int32
+            minimum: 1
+            maximum: 100
+            default: 10
+      responses:
+        "200":
+          $ref: "#/components/responses/200SourceArray"
+        "400":
+          $ref: "#/components/responses/400"
+        "401":
+          $ref: "#/components/responses/401"
+        "422":
+          $ref: "#/components/responses/422"
+        "500":
+          $ref: "#/components/responses/500"
+    post:
+      summary: Creates a new source
+      operationId: createSource
+      requestBody:
+        description: Source to add
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/NewSource"
+      responses:
+        "201":
+          $ref: "#/components/responses/201Source"
+        "400":
+          $ref: "#/components/responses/400"
+        "401":
+          $ref: "#/components/responses/401"
+        "422":
+          $ref: "#/components/responses/422"
+        "500":
+          $ref: "#/components/responses/500"
+  /sources/{id}:
+    parameters:
+      - name: id
+        in: path
+        description: The source ID
+        required: true
+        schema:
+          type: string
+          pattern: '^[a-f\d]{24}$'
+    get:
+      summary: Gets a specific source
+      operationId: getSource
+      responses:
+        "200":
+          $ref: "#/components/responses/200Source"
+        "400":
+          $ref: "#/components/responses/400"
+        "401":
+          $ref: "#/components/responses/401"
+        "404":
+          $ref: "#/components/responses/404"
+        "422":
+          $ref: "#/components/responses/422"
+        "500":
+          $ref: "#/components/responses/500"
+    put:
+      summary: Updates a specific source
+      operationId: updateSource
+      requestBody:
+        description: Source to update
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/Source"
+      responses:
+        "200":
+          $ref: "#/components/responses/200Source"
+        "400":
+          $ref: "#/components/responses/400"
+        "401":
+          $ref: "#/components/responses/401"
+        "404":
+          $ref: "#/components/responses/404"
+        "422":
+          $ref: "#/components/responses/422"
+        "500":
+          $ref: "#/components/responses/500"
+    delete:
+      summary: Deletes a specific source
+      operationId: deleteSource
+      responses:
+        "204":
```

Comment:
I found this with the data service too -- currently it looks like this is returning a 200 OK for success instead of 204, and it's returning the Source object with the 200:

https://github.com/open-covid-data/healthmap-gdo-temp/blob/master/verification/curator-service/api/src/controllers/sources.ts#L158

you can see my change for this in the data service here: https://github.com/open-covid-data/healthmap-gdo-temp/pull/197/files

Reply:
Ah, thanks. Done.